develop/2017-06-20-rodolfoviana-expenses-on-car-rental.py

```diff
@@ -0,0 +1,143 @@
+                             'document_id': np.str},
+                      low_memory=False)
+
+
+# #### Selecting term, subquota
+# 
+# There are 14,263 reimbursements since Jan. 2015. They sum up to R$ 60,373,960.80.
+
+# In[2]:
+
+data = data[data['year'] >= 2015]
+data = data[data['subquota_description'] == 'Automotive vehicle renting or charter']
+data['cnpj_cpf'] = data['cnpj_cpf'].str.replace(r'[\.\/\-]', '')
+data.subquota_description.value_counts()
+
+
+# In[3]:
+
+data.net_values.sum()
+
+
+# In[4]:
+
+congressperson_list = data[['congressperson_name', 
+                            'congressperson_id', 
+                            'net_values', 
+                            'month', 
+                            'year', 
+                            'issue_date', 
+                            'document_id',
+                            'cnpj_cpf']]
+
+
+# In[5]:
+
+congressperson_expenses = congressperson_list.groupby(['congressperson_name', 
+                                                       'year', 
+                                                       'month', 
+                                                       'issue_date', 
+                                                       'document_id']).agg({'net_values':sum})
+
+congressperson_expenses.head(20)
+
+
+# #### Getting companies dataset, excluding those from DF, merging with reimbursements dataset
+# 
+# There are 10,945 companies outside DF. The receipts sum up to R$ 49,156,001.56.
+
+# In[6]:
+
+companies = pd.read_csv('../data/2017-05-21-companies-no-geolocation.xz', low_memory=False)
+companies = companies[companies['state'] != 'DF']
+companies['cnpj'] = companies['cnpj'].str.replace(r'[\.\/\-]', '')
+
+
+# In[7]:
+
+dataset = pd.merge(data, companies, how='inner',
+                   left_on='cnpj_cpf', right_on='cnpj')
+
+
+# In[8]:
+
+congressperson_expenses_dataset = dataset.groupby(['congressperson_name', 
+                                                    'year', 
+                                                    'month', 
+                                                    'issue_date',
+                                                    'cnpj',
+                                                    'name',
+                                                    'city',
+                                                    'state_y',
+                                                    'document_id']).agg({'net_values':sum})
+full_report = congressperson_expenses_dataset.reset_index()
+
+
+# In[9]:
+
+full_report.name.count()
+
+
+# In[10]:
+
+full_report.net_values.sum()
+
+
+# #### Getting outliers
+# 
+# Although Rosie considers mean value + three times the std value to point outliers, here we consider mean value + twice the std value. This is due to the subquota category -- different from other categoris (i.e. taxi, food, hotel), big money is spent on car rental, and work with 3 x std value would let suspect receipts pass.
```

Comment:
Hope the paper I linked can help you understand if 2, 3 or any other number of deviations from the mean is good enough to the problem being explored here.